packages/arb-token-bridge-ui/src/util/TokenListUtils.ts

```diff
@@ -34,39 +36,39 @@ export const BRIDGE_TOKEN_LISTS: BridgeTokenList[] = [
     isArbitrumTokenTokenList: true
   },
   {
-    id: 1,
+    id: '1',
     originChainID: ChainId.ArbitrumOne,
     url: 'https://tokenlist.arbitrum.io/ArbTokenLists/arbed_arb_whitelist_era.json',
     name: 'Arbitrum Whitelist Era',
     isDefault: true,
     logoURI: ArbitrumLogo
   },
   {
-    id: 2,
+    id: '2',
```

Comment:
we should unify the id syntax and use `${chain.chainId}_native` or `${chain.chainId}_uniswap` or `_cmc` etc, then tighten the type